dist/testbed-support/makefile.iotlab.single.inc.mk

```diff
@@ -0,0 +1,190 @@
+# Use a single iot-lab node with RIOT
+# ===================================
+#
+# Supported:
+#  * flash
+#  * reset
+#  * term
+#
+# Tested on m3/a8-m3/wsn430/samr21/arduino-zero nodes
+#
+# It can be run:
+# * From your computer by setting IOTLAB_NODE to the full url like
+#   m3-380.grenoble.iot-lab.info or a8-1.grenoble.iot-lab.info
+# * From IoT-LAB frontend by setting IOTLAB_NODE to the short url like
+#   m3-380 or a8-1
+#
+# # Usage
+#
+#   make BOARD=iotlab-m3 IOTLAB_NODE=m3-380.grenoble.iot-lab.info flash term
+#
+# It is the user responsability to start an experiment beforehand
+#
+# If the user has multiple running experiments, the one to use must configured
+# with the `IOTLAB_EXP_ID` setting
+#
+# Prerequisites
+# -------------
+#
+# * Install iotlab-cli-tools: https://github.com/iot-lab/cli-tools
+# * Install iotlab-ssh-cli-tools: https://github.com/iot-lab/ssh-cli-tools
+#   * Can be done using given `requirements.txt`
```

Comment:
This is unclear

Reply:
Problem, do I explain with `sudo pip install -r` or the right way `pip install --user -r` and then expect them to setup their `PATH` properly ? That is why I linked with the official documentation for it.

It was not explained how to install the cli-tools in the other one either.

That makes me think I did not put information in the `README.iotlab`.

Reply:
Indeed, there's no installation section in the iotlabcli readme but there's one in the iotlabsshcli. Maybe you could just link to the related cli-tools and ssh-cli-tools tutorials on the IoT-LAB website ?

Reply:
I removed the mention on requirements.txt entirely.
My official documentation is use the IoT-LAB github documentation to install, I only added requirements.txt as version reference and for advanced users.